lib/tasks/upgrade.rake

```diff
@@ -696,6 +701,11 @@ namespace :upgrade do
     Role.reviewer.destroy_all
   end
 
+  desc "Sets the new managed flag for all existing Orgs to managed = true"
+  task default_orgs_to_managed: :environment do
+    Org.all.update_all(managed: true)
```

Comment:
Should we have an exception for the `other org` or is that invalidated by your other ROR PR?